provider/workspace.go

```diff
@@ -83,6 +91,11 @@ func workspaceDataSource() *schema.Resource {
 				Computed:    true,
 				Description: "The access port of the Coder deployment provisioning this workspace.",
 			},
+			"prebuild_count": {
```

Comment:
`is_prebuild` & `prebuild_count` looks the same to me, what's the point to have both?

Reply:
This seems to be duplication, yes. We could probably get rid of "is_prebuild" and just check the count. Looking at the rest of the code, we are following the pattern that was set by the "transition" and "start_count" parameters. They have the same relationship.

I'm not sure whether to remove "is_prebuild" or keep it.

Reply:
I don't like neither `prebuild_count` nor `start_count`. But probably there is some reason we're using, I'm not aware of.
My guess is we're using `start_count` to map transition statuses, like:
- stop: 0
- start: 1
etc...

Also looks like there is some duplication here as well:
- start_count
- transition

But I don't know why it's needed.

We can ask original author of this approach with `start_count`, `transition`?
Maybe it was done in a rush, and there is no sense to continue using this approach.